wikiViewer.js

```diff
@@ -0,0 +1,64 @@
+//opening the page
+$(document).ready(function() {
+
+    //clicking on search
+
+    $('#searchButton').click(function() {
+
+
+        //define variables
+
+        var searchTerm = $('#searchTerm').val();
+
+        //API url with serach implement
+
+        var url = "https://en.wikipedia.org/w/api.php?action=opensearch&search=" + searchTerm + "&format=json&callback=?";

```

Comment:
czesc stala jest raczej zmienna na aplikacje, wiec powinna byc w scope wyzej. dlaczego? gdybys miala to w paru miejscach a pozniej okazaloby sie, ze adres API sie zmienia to musialabys to zmieniac w paru miejscach (dlatego tez dobrze jakbys tez cale budowanie urla wyodrebnila do funkcji pomocniczej + skorzystala z configa w realnej, wiekszej aplikacji :))